backtester/report/tpl.gohtml

```diff
@@ -0,0 +1,712 @@
+					<thead>
+					<th>Exchange</th>
+					<th>Asset</th>
+					<th>Currency</th>
+					<th>Starting funds</th>
+					<th>Resulting funds</th>
+					<th>Did it make profit?</th>
+					<th>Did it beat the market?</th>
+					<th>Strategy Movement</th>
+					<th>Market Movement</th>
+					</thead>
+					<tbody>
+                    {{ range $exchange, $unused := .Statistics.ExchangeAssetPairStatistics}}
+                        {{ range $asset, $unused := .}}
+                            {{ range $pair, $unused := .}}
+								<tr>
+									<td>{{ $exchange}}</td>
+									<td>{{ $asset}}</td>
+									<td>{{ $pair}}</td>
+									<td>${{ printf "%.2f" .FinalHoldings.InitialFunds }} {{.FinalHoldings.Pair.Quote}}</td>
+									<td>${{ printf "%.2f" .FinalHoldings.TotalValue }} {{ .FinalHoldings.Pair.Quote}}</td>
+									<td>{{ gt  .FinalHoldings.TotalValue .FinalHoldings.InitialFunds }}</td>
+									<td> {{ gt .StrategyMovement  .MarketMovement }}</td>
+									<td>{{ printf "%.2f" .StrategyMovement }}%</td>
+									<td>{{ printf "%.2f" .MarketMovement }}%</td>
+								</tr>
+                            {{end}}
+                        {{end}}
+                    {{end}}
+					</tbody>
+				</table>
+			</div>
+
+		</div>
+	</div>
+	<div >
+		<div class="card card-cascade narrower">
+			<div class="view view-cascade bg-primary">
+				<h2  class="px-4 card-header-title white-text">Config</h2>
+			</div>
+		</div>
+	</div>
+	<div >
+		<div class="card card-cascade narrower">
+			<div class="view view-cascade bg-secondary">
+				<h2  class="px-4 card-header-title white-text">Strategy Settings</h2>
+			</div>
+			<div class="card-body card-body-cascade ">
+				<table class="table table-hover table-bordered table-striped">
+					<tbody>
+					<tr>
+						<td><b>Strategy name</b></td>
+						<td>{{.Config.StrategySettings.Name}}</td>
+					</tr>
+					<tr>
+						<td><b>Is multi currency</b></td>
+						<td>{{.Config.StrategySettings.SimultaneousSignalProcessing}}</td>
+					</tr>
+					<tr>
+						<td><b>Custom settings</b></td>
+						<td>{{.Config.StrategySettings.CustomSettings}}</td>
+					</tr>
+					</tbody>
+				</table>
+			</div>
+		</div>
+	</div>
+	<div >
+		<div class="card card-cascade narrower">
+			<div class="view view-cascade bg-info">
+				<h2  class="px-4 card-header-title white-text">Currency Settings</h2>
+			</div>
+			<div class="card-body card-body-cascade ">
+				<table class="table table-hover table-bordered table-striped">
+					<thead>
+					<th>Exchange Name</th>
+					<th>Asset</th>
+					<th>Currency Base</th>
+					<th>Currency Quote</th>
+					<th>Initial Funds</th>
+					<th>Buy side Min Amount</th>
+					<th>Buy side Max Amount</th>
+					<th>Buy side Max Total</th>
+					<th>Sell side Min Amount</th>
+					<th>Sell side Max Amount</th>
+					<th>Sell side Max Total</th>
+					<th>Min Slippage Percent</th>
+					<th>Max Slippage Percent</th>
+					<th>Maker Fee</th>
```

Comment:
![image](https://user-images.githubusercontent.com/31364354/107729704-b1515300-6d45-11eb-8a9d-bd87889b6404.png)
Maker fee shouldn't be higher than taker. Maybe somewhere we need to add a check

Reply:
Since its user supplied, I wasn't too worried, I also do not utilise maker fees yet, I'll remove taker from displaying in reports.

With regards to adding a check, in thescenario that taker is bigger than maker, what should occur? Maker could equal taker, or I can stop the application from running which seems extreme.

Reply:
Probably a warning should suffice